ppoprf/src/ppoprf.rs

```diff
@@ -115,7 +141,8 @@ impl Server {
         }
     }
 
-    pub fn eval(&self, p: &CompressedRistretto, md_idx: usize, verifiable: bool) -> Evaluation {
+    pub fn eval(&self, p: &Point, md_idx: usize, verifiable: bool) -> Evaluation {
```

Comment:
So is the `Point` struct the public facing object that clients send to the randomness server?

Reply:
It's a wrapper the client and server can both use for serialisation. CompressedRistretto is already a public type, but we want to use serde to control the format the curve points are sent/received in.

Since CompressedRistretto comes from a foreign crate we can't use the  derive macro directly, so we wrap it in a local type and derive (de)serialisation from there. The other alternative is to implement a custom serialisation function for the (public) type, but that's more code.

This could go in the example server and client; in theory it's up to them to agree what the serialised format should look like. However, I think it's fine for `ppoprf::Server` implementation to have an opinion to encourage standardisation, and then it's easier to share the implementation between all.